### DISCUSSION TITLE
How to have Vector only pickup older files in Source (files)

### DISCUSSION THREAD

**Discussion by Trowa929** [vectordotdev/vector#21862] [General]
How can I configure vector to only grab day old files or older in the source portion of conifg (file)

I see in the docs there is an option for ignore_older_secs -> Is there a way to do the opposite?

Thanks,
Sam

**Comment by pront** [comment]
I don't think the `file` source supports that. Feel free to open a [feature request](https://github.com/vectordotdev/vector/issues/new?assignees=&labels=type%3A+feature&projects=&template=feature.yml). I also think there's room for a more generic solution with VRL conditions here but that's a bigger undertaking. 

As a workaround you could organize you files in folders based on time intervals and then use the https://vector.dev/docs/reference/configuration/sources/file/#exclude globs.
